### ISSUE TITLE
Register

### ISSUE THREAD

**Issue by ilkerarslan** [ilkerarslan/NovaML.jl#27]
@JuliaRegistrator register

  Release notes:

  ## Breaking changes

  - None intended; version bumped to 0.5.0 for the new alpha release.

  ## New features

  - Add `GradientBoostingRegressor` in `NovaML.Ensemble` — gradient boosting for regression tasks using decision tree base learners.
  - Support `squared_error`, `absolute_error`, and `huber` losses.
  - Add common gradient boosting options including `learning_rate`, `n_estimators`, `subsample`, `max_depth`, `random_state`, `warm_start`, and early stopping via
  `n_iter_no_change`.
  - Expose fitted attributes such as `estimators_`, `train_score_`, `feature_importances_`, and `n_estimators_`.

  ## Fixes

  - None.
